### PR TITLE
adds section on parameters with links to spreadsheet

### PR DESCRIPTION
**Pull request by cjakeman**
Also adds cross-references in several chapters to this section.
The spreadsheet is hosted on-line and updated by John Brady.
Forum post http://www.elvastower.com/forums/index.php?/topic/33504-engwag-values-the-definitive-list/page__view__findpost__p__267875

### CODE REVIEW

**Review by peternewell — APPROVED** [review]
Approved.

Thanks for supporting John in this work.

File: .gitignore [hunk 1]
```diff
@@ -360,3 +360,4 @@ healthchecksdb
 
 # Backup folder for Package Reference Convert tool in Visual Studio 2017
 MigrationBackup/
+Source/Documentation/Manual/.vscode/settings.json
```
Comment (YoRyan) [inline .gitignore:363]: A personal preference snuck in here

**Review by YoRyan — CHANGES_REQUESTED** [review]

**Comment by YoRyan** [comment]
Also, would it be possible to have John supply a release copy that we can distribute on our own website or with the manual files?

The last thing I'd like to see is a dead tinyurl link about 5 or so years down the line.

File: .gitignore (hunk 1 above)
Comment (cjakeman) [reply .gitignore]: Thanks, Ryan

I've removed that now.

Best wishes,

Chris

File: Source/Documentation/Manual/developing.rst [hunk 2]
```diff
@@ -47,6 +47,43 @@ Activities
...
++------------------------------+-----------------------------+
+| signal scripts               |        sigscr.dat           |
++------------------------------+-----------------------------+
+| sound management             |        sms                  |
++------------------------------+-----------------------------+
+| train timetable              |        timetable-or         |
++------------------------------+-----------------------------+
+
+The complete list is very extensive and is documented in `an on-line spreadsheet at http://tinyurl.com/or-parameters-excel
+<http://tinyurl.com/or-parameters-excel>`_
```
Comment (YoRyan) [inline Source/Documentation/Manual/developing.rst:83]:
```suggestion
The complete list is very extensive and is documented in an online spreadsheet at `tinyurl.com/or-parameters-excel
<https://tinyurl.com/or-parameters-excel>`_.
```
This link could be cleaner (and should use HTTPS).

**Review by YoRyan — CHANGES_REQUESTED** [review]

**Review by YoRyan — APPROVED** [review]
Thanks, Chris. Approved.

**Pull request merged**